src/assets/text/monitoringText.js

```diff
@@ -1,24 +1,26 @@
-        title4: "Implications for Temperature Modeling",
-        paragraph14: "Data sparsity and variability in stream temperature across the network limit our ability to answer questions like: How far downstream will a cold water release affect temperature?  How has fish habitat changed through time?  Will we have enough cold water in the reservoir in 50 years?",
-        paragraph15: "These challenges are not unique to the Delaware River Basin. In fact, relative to the rest of the United States, the Delaware River Basin is one of the most highly monitored basins in the country.",
-        paragraph16: "How can we create models that are robust to a diversity of temperature dynamics, in places and times where we don't have a lot of information? Visit the Model chapter to learn about innovative modeling techniques to predict water temperature."
+        paragraph11: "To show how gaps in temperature observations in space and time might impact our understanding of stream temperature dynamics, we can represent the availability of data as a matrix with two axes: space and time. Some streams are monitored daily, others at regular intervals, and some sporadically or not at all. It is more difficult to predict how temperature in a stream will fluctuate when we don't have much data to go on.",
+        title4: "Water Temperature Variation",
+        paragraph12: "Another complexity of predicting temperature dynamics in the basin is that not all streams behave the same way. Features of the landscape can have large effects on water temperature that result in different temperature patterns. Groundwater inputs, reservoirs, and urban areas all have distinct temperature signatures.",
+        paragraph13: "If we look at a single year of data, we can see that stream temperatures are not uniform throughout the basin.",
+        paragraph14: "In the summer of 2019, an urban reach in Philadelphia was the warmest stream reach in the basin, while a reach below the Beltzville Resevoir was one of the coolest. These reaches aren't that far from each other as the crow flies, but they experience very different seasonal temperature variations.",
+        paragraph15: "The matrix below shows a year of actual temperature data from all sites on medium to large streams in the Delaware River Basin.",
+        paragraph16: "At this point, we know that stream temperatures on any given day vary throughout the basin and that the amount of data we have to work with varies in time and space. To predict what the temperature will be in a given stream, or how temperatures will change over time, we can use a model. But how can we create robust models that account for diverse temperature dynamics and make accurate predictions in places and times where we don't have a lot of information? Visit the Model chapter to learn about innovative modeling techniques to predict water temperature."
```

Comment:
For next steps, we should create a class to hide elements we don't want shown when we first publish the page. We could add it to this last sentence as well as the links etc to make them inactive.